--!strict
return {
	Quest1 = {
		"Time for your first punishing quest",
		"You will have to outlaster your opponents",
		"Stay on your feet as the speed increases",
		"Only the top 25 will survive!"
	},
	TeamPick = {
		"The survivors will now form teams!",
		"Top players become team leaders...",
		"Choose your teammates wisely!"
	},
	Quest2 = {
		"Time for some teamwork!",
		"Collect blocks and circles from your team's area",
		"Build your path to victory!",
		"Get your whole team to the finish line to pass...."
	},
	Quest3 = {
		"Keep that momentum going!",
		"Work together to move the boulder",
		"Get to Station 2, then back to Station 1",
		"Only the fastest teams will survive..."
	},
	Quest4 = {
		"The pressure is mounting!",
		"Grab sand bags from your pile",
		"Stack them high on your platform",
		"Most bags in 100 seconds wins!"
	},
	Quest5V1 = {
		"Time to eliminate your own teammates",
		"Run on the stairs as long as possible...",
		"First to touch the ground fails!"
	},
	Quest5V2 = {
		"No time to rest!",
		"Run across the track and tag the player in front of you....",
		"Final 2 players pass"
	},
	Quest5V3 = {
		"The grand finale!",
		"Do the most pull ups in 100 seconds in order to win",
		"Who will be the ultimate champion?"
	}
} :: {[string]: {string}}